common/app/model/content.scala

```diff
@@ -514,21 +514,27 @@ object Video {
     val elements = content.elements
     val section = content.metadata.section
     val id = content.metadata.id
-    val source: Option[String] = elements.videos.find(_.properties.isMain).flatMap(_.videos.source)
+    val mainVideo: Option[VideoElement] = elements.videos.find(_.properties.isMain)
+    val source: Option[String] = mainVideo.flatMap(_.videos.source)
 
     val javascriptConfig: Map[String, JsValue] = Map(
       "isPodcast" -> JsBoolean(content.tags.isPodcast),
       "source" -> JsString(source.getOrElse("")),
-      "embeddable" -> JsBoolean(elements.videos.find(_.properties.isMain).map(_.videos.embeddable).getOrElse(false)),
-      "videoDuration" -> elements.videos.find(_.properties.isMain).map{ v => JsNumber(v.videos.duration)}.getOrElse(JsNull))
-
-    val optionalOpengraphProperties = if(content.metadata.webUrl.startsWith("https://")) Map("og:video:secure_url" -> content.metadata.webUrl) else Nil
-    val opengraphProperties = Map(
-      "og:type" -> "video",
-      "og:video:type" -> "text/html",
-      "og:video" -> content.metadata.webUrl,
-      "video:tag" -> content.tags.keywords.map(_.name).mkString(",")
-    ) ++ optionalOpengraphProperties
+      "embeddable" -> JsBoolean(mainVideo.map(_.videos.embeddable).getOrElse(false)),
+      "videoDuration" -> mainVideo.map{ v => JsNumber(v.videos.duration)}.getOrElse(JsNull))
+
+    val opengraphProperties = mainVideo.flatMap { _.videos.largestVideo }.map { videoAsset =>
```

Comment:
That would share the largest video. We can probably find a better suited size to share on FB but I want to first make sure this is the correct solution before to introduce any complexity